table-attributes.md

```diff
@@ -73,47 +75,49 @@ alter table t partition p attributes[=]'key=value1';
 
 ### 使用方法
 
+使用方法如下，其中 `t` 为所要修改的表名，`p` 为所要修改的分区名。
+
 + 禁止属于某个表的 Region 被合并
 
     ```sql
-    alter table t attributes[=]'merge_option=deny';
+    ALTER TABLE t ATTRIBUTES 'merge_option=deny';
     ```
 
 + 允许属于某个表的 Region 被合并
 
     ```sql
-    alter table t attributes[=]'merge_option=allow';
+    ALTER TABLE t ATTRIBUTES 'merge_option=allow';
     ```
 
 + 重置某个表的属性
 
     ```sql
-    alter table t attributes[=]default；
+    ALTER TABLE t ATTRIBUTES default;
     ```
 
 + 禁止属于某个分区的 Region 被合并
 
     ```sql
-    alter table t partition p attributes[=]'merge_option=deny';
+    ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=deny';
     ```
 
 + 允许属于某个分区的 Region 被合并
 
     ```sql
-    alter table t partition p attributes[=]'merge_option=allow';
+    ALTER TABLE t PARTITION p ATTRIBUTES 'merge_option=allow';
     ```
 
 + 查看所有配置了 `merge_option` 属性的表或分区
 
     ```sql
-    select * from information_schema.attributes where attributes like '%merge_option%';
+    SELECT * FROM information_schema.attributes WHERE attributes LIKE '%merge_option%';
```

Comment:
maybe upper cases for "attributes" here as well? @dveeden

Reply:
`attributes` here is the name of a column (an identifier) and not a keyword. So changing this to uppercase is not required.

Reply:
```suggestion
    SELECT * FROM information_schema.attributes WHERE ATTRIBUTES LIKE '%merge_option%';
```
have updated according to the newly updated English PR